.gitignore

```diff
@@ -0,0 +1,6 @@
+**/policygen
+**/tfengine
+**/tfimport
+!**/policygen/
```

Comment:
Put a comment above binaries and above these dir exceptions so users can know what each block is doing

Reply:
Done.